files/routes/dev.py

```diff
@@ -0,0 +1,17 @@
+from secrets import token_hex
+from flask import session, redirect, request
+
+from files.helpers.const import PERMS
+from files.helpers.get import get_user
+from files.helpers.wrappers import admin_level_required
+from files.__main__ import app
+
+@app.post('/dev/sessions/')
+@admin_level_required(PERMS['DEBUG_LOGIN_TO_OTHERS'])
+def login_to_other_account(v):
+	u = get_user(request.values.get('username'))
+	session.permanent = True
+	session["lo_user"] = u.id
+	session["login_nonce"] = u.login_nonce
+	session["session_id"] = token_hex(49)
+	return redirect('/')
```

Comment:
I'm worried that this is guarded from release builds *only* by the nonexistence of an import, which seems like something that could be accidentally imported from somewhere else and we wouldn't realize it.. Could I get a `if not app.debug: return` at the top, or something like that?

Reply:
:marseythumbsup: